### ISSUE TITLE
Does not Support for HTML5

### ISSUE THREAD

**Issue by developer100rbh** [RodionGork/JsTagSphere#2]
Hello, I need to implement this Library in one of my Android Project, but this is too heavy for WebView and does not work on HTML5. I Need HTML5 support with this. Thanks!

**Comment by jkjustjoshing** [comment]
What would "HTML5" support look like to you?

**Comment by RodionGork** [comment]
@jkjustjoshing perhaps it is about words "no HTML5" in `index.html` page. I'm afraid I've created this little thing some years ago and don't remember what precisely it meant. Probably not that it won't work with HTML5 :)

I'll try to check.

**Comment by jkjustjoshing** [comment]
Yeah, to me that means "it works on older browsers", not "it won't work in newer browsers".

**Comment by RodionGork** [comment]
@developer100rbh & @jkjustjoshing - thanks for bringing attention to it. I see now that at least in some cases adding `<!doctype html>` really breaks it. I'll try to take care for it :)

**Comment by RodionGork** [comment]
It appeared @tdbts have solved this 4 years ago and proposed PR. I'm very sorry for not merging it in time. For now it is merged (though I think I created some mess with branches off-shooting gh-pages beforehand). But live demo page is now at github pages (see readme) - and it is in HTML5 style.
